packages/medusa/src/models/order-edit.ts

```diff
@@ -72,6 +71,9 @@ export class OrderEdit extends BaseEntity {
   @Column({ type: resolveDbType("timestamptz"), nullable: true })
   canceled_at?: Date
 
+  @OneToMany(() => LineItem, (lineItem) => lineItem.order_edit)
+  items: LineItem[]
```

Comment:
question: do we keep the `changes` property as well as the `items` property in the db or do we let this update be the new source of truth about what the order should look like after confirmation? 
keeping the `changes` seems like it will be double work and storage for the edits.

@srindom

Reply:
As we will use the cloned items to be attached to the order and detached the order items, the changes will not be used for that too.
If we wanted to compute the change without having to store them we could go with the following:
- ITEM_ADD: item with order_edit_id and no original_item_id
- ITEM_UPDATE: item with order_edit_id and original_item_id
- ITEM_REMOVE: all items that have an order id but no clone anymore

Reply:
I think we should stick with having the changes stored as well - this way we have some way of auditing (and displaying history in admin).

Reply:
But changes will in this case not be tied to business logic